charts/osm/templates/osm-deployment.yaml

```diff
@@ -26,6 +26,13 @@ spec:
         prometheus.io/port: '9091'
     spec:
       serviceAccountName: {{ .Release.Name }}
+      {{- if not (.Capabilities.APIVersions.Has "security.openshift.io/v1") }}
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 3000
+        fsGroup: 2000
+        supplementalGroups: [5555]
```

Comment:
Do all deployments use the same user and group numbers?

Reply:
Yes, these seem to be standard numbers. I'll let @ritazh comment

We could pull these out into a template but I felt like it was a bit more readable to be explicit here.

Reply:
I'd suggest to template it if is meant to be common, and also document their significance.

Reply:
I'll do this in a follow up. Still have to configure PSP for jaeger and fluentbit in a separate PR

Reply:
I agree with documenting the significance, but not sure templating would make this any better. You'd have to go elsewhere to find out what the exact value of this is. If we don't expect this to change over time - there should be no need to template it.

Reply:
Templating it could mean 2 things here:
1. Define these constants in 1 file and include them where necessary
2. Use variables for the specified field values

I was referring to 1.